Makefile.rules

```diff
@@ -535,6 +535,7 @@ clean:
 	rm -f bin/gap.sh sysinfo.gap*
 	rm -f gap$(EXEEXT) gac ffgen
 	rm -f libgap.la
+	rm -f libgap.pc
```

Comment:
Another question (*not* a change request!): is the file `libgap.pc` ever useful in the non-installed version? If not, then we could also omit most of the rules for building/removing it, and only create it on the fly in the `install-libgap` target (again, not requesting that you do so)

Reply:
Why would you want to fill in a template not by `./configure`, but in a custom way? What does it buy you, except increase in unreadability?

Reply:
Do you consider `config.status`, or the for that matter `configure`, readable? There is a ton of "magic" there. And to support this one file, code has to be changed in a bunch of places.

The "unreadable code" would look like this:
```
define libgap_pc
prefix=@prefix@
exec_prefix=@exec_prefix@
libdir=@libdir@
includedir=@prefix@/include

Name: GAP
Description: Library containing the kernel of GAP, a computer algebra system
... the rest
endef
export libgap_pc
...
	$(INSTALL) -d -m 0755 $(DESTDIR)$(libdir)/pkgconfig
	@echo "$$libgap_pc" > $(DESTDIR)$(libdir)/pkgconfig/libgap.pc
```

Reply:
no, these are not readable - but `configure.ac` is readable, and `Makefile.rules` is readable.

Reply:
`autoconf` comes with a macro to fill in templates, why the itch to re-do manually?

Reply:
You haven't answered my question yet:  is the file `libgap.pc` ever useful in the non-installed version?

I am going to assume the answer is "no".